.circleci/config.yml

```diff
@@ -13,42 +13,33 @@ commands:
       - run: brew install protobuf cmake ccache libtool libspatialite pkg-config luajit curl wget czmq lz4 spatialite-tools unzip
       - run: pip3 install requests shapely conan
       - run: git clone https://github.com/kevinkreiser/prime_server --recurse-submodules && cd prime_server && ./autogen.sh && ./configure && make -j8 && make install
+  install_linux_dependencies:
+    steps:
+      - run: bash ./scripts/install-linux-deps.sh
 
 jobs:
   build-docker-images:
-    docker:
-      - image: docker:18.06.3-ce-git
+    machine:
+      image: ubuntu-2004:2022.07.1
```

Comment:
we can use the circleci provided image so it doesn't download: circleci.com/developer/machine/image/ubuntu-2204#how-this-image-works

Reply:
switch to `current` ubuntu tag